src/operations/mutations/ProjectMutations.js

```diff
@@ -18,6 +18,25 @@ export const ADD_PROJECT = gql`
     }
 `
 
+export const SYNC_GITHUB_ISSUES = gql`
+    mutation syncGithubIssues($project_id: Int!) {
+        syncProjectIssues(project_id: $project_id) {
+            id
+        }
+    }
+`
+
+export const SYNC_PROJECT_GITHUB_CONTRIBUTORS = gql`
+    mutation syncProjectGithubContributors($project_id: Int!, $github_personal_key: String){
+        syncProjectGithubContributors(project_id: $project_id, github_personal_key: $github_personal_key) {
```

Comment:
@sofiarm21 not sure what the impact is of this but I noticed in the ProjectType we have `github_personal_key` defined but in the ProjectResolvers we have `args.githubPersonalKey` so this discrepancy will fail every time but since it has a fallback (https://github.com/setlife-network/trinary/blob/b376789381faf2b47a1ea9854a4f4826d4288b8d/api/schema/resolvers/ProjectResolver.js#L556) it probably doesn't actually cause an error.

Merging for now but as a separate issue it's probably worth doing a project-wide search for `githubPersonalKey` just to refactor/remove after making sure it is needed and works properly in the cases it should be